Automattic/wp-calypso
# Launchpad: Add the Share Site Modal component in the package

Related to #

## Proposed Changes

* Moves the Share Site Modal to the Launchpad package. This will allow us to reuse the component in other contexts, such as the Navigator.
* This PR will also resolve a weirdness with the tasks being out of order.

## Testing Instructions



Since I did some core changes that relate to the pre & post-launch Launchpads, we should test the pre & post-launchpad and navigator.

Use the Calypso Live link below or apply this PR to your local environment

#### Testing the Pre-Launch Launchpad

* Create a new site with the build intent
* Go through the flow until you reach the fullscreen launchpad
* Skip the fullscreen launchpad by click on the top right corner
* You should be redirected to the customer home
* Make sure you can complete all the tasks and launch your site

#### Testing the Post-Launch Launchpad

* With the site you just created above
* You should see the post-launch Launchpad
* Try to complete all the tasks

#### Testing the Navigator

* On a site with the "build" intent, navigate to `/home/:siteSlug?flags=launchpad/navigator`
* Click on the progress ring on the master bar
* Then, click on the "Share your site" task
* Make sure the modal work as expected(use the copy button)




## Pre-merge Checklist
